### DISCUSSION TITLE
Generator improvements

### DISCUSSION THREAD

**Discussion by Aides123** [BG-Software-LLC/SuperiorSkyblock2#290] [Denied Feature Requests] [answered]
Hi, how about upgrading the generators?
The current system works great but 1 thing is missing.

It is impossible to make separate levels to upgrade the generator per world by which the player goes by turn.
Separate levels cannot be created to upgrade the generator to the world and the player must unlock all of them.
So I propose a separate configuration per world - world / nether / end
I know, I know it is possible, but as I did above, the player has to unlock everything, not the selected generators for a given world.

**Comment by OmerBenGera** [comment, marked as answer]
You can make different generators for different worlds, so I am not sure what you're requesting here.

**Reply by haroldrc** [reply]
how do i make a generator for nether?

**Reply by OmerBenGera** [reply]
> 
> 
> how do i make a generator for nether?

Add the rates you want under the nether section in the generators section.
For more information regarding how to do that, you can visit the wiki of the plugin at https://bg-software.com/superiorskyblock
